packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -821,6 +822,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return this._summarizer;
     }
 
+    private readonly metadata: IContainerRuntimeMetadata = {summaryFormatVersion: 1, message: undefined};
```

Comment:
Can we somehow avoid having to update this version in two places (whenever we need to bump it)?
See 
export interface IContainerRuntimeMetadata {
    readonly summaryFormatVersion: 1;